pkcs7.go

```diff
@@ -864,13 +934,19 @@ func encryptDESCBC(content []byte) ([]byte, *encryptedContentInfo, error) {
 //     ContentEncryptionAlgorithm = EncryptionAlgorithmAES128GCM
 //
 // TODO(fullsailor): Add support for encrypting content with other algorithms
-func Encrypt(content []byte, recipients []*x509.Certificate) ([]byte, error) {
+func Encrypt(content []byte, recipients []*x509.Certificate, opts ...Option) ([]byte, error) {
```

Comment:
This changes the function signature, but is still backwards compatible depending on how Encrypt is called. 
`pkcs7.Encrypt(content, recipients)` would still compile, but if someone created a callback, or interface their code would break. 

Possible solutions:
1) break compatibility and create a release with a new git tag
2) create a new EncryptWithOptions function
3) use global variables (personally I would avoid but I can work with that option if required)